.meta.toml

```diff
@@ -6,4 +6,9 @@ template = "default"
 commit-id = "cfffba8c"
 
 [pyproject]
-dependencies_ignores = "['lxml']"
+dependencies_ignores = "['lxml', 'plone.app.vocabularies']"
```

Comment:
I see that `lxml` is a hard dependency, so ignoring it is probably still fine, as we will anyway get it from plenty of other packages, but there is no problem adding it, or? 🤔 

For `plone.app.vocabularies`, we should at least document that there is a circular dependency, so we can work to fix it.

Created at https://github.com/plone/Products.CMFPlone/issues/3816